megamek/src/megamek/common/Entity.java

```diff
@@ -6225,12 +6230,30 @@ public void assignAMS(Vector<WeaponHandler> vAttacks) {
                     vAttacksInArc.addElement(wr.waa);
                 }
             }
-            // find the most dangerous salvo by expected damage
+            //AMS Bays can fire at all incoming attacks each round
```

Comment:
I may be off on this, because I don't know the rules, but should AMS bays and PD bays work the same in atmosphere as in space?  Right now, that's how the code is written.

Reply:
This is correct RAW. I only changed the behavior to work properly with our optional 'use individual weapons for grounded dropships' rule. TW/SO assume that you always use bay weapons for dropships for ease of play.